tree/dataframe/test/dataframe_nodes.cxx

```diff
@@ -11,7 +11,7 @@ TEST(RDataFrameNodes, RSlotStackCheckSameThreadSameSlot)
 {
    unsigned int n(7);
    ROOT::Internal::RDF::RSlotStack s(n);
-   EXPECT_EQ(s.GetSlot(), s.GetSlot());
+   EXPECT_EQ(s.GetSlot()-1, s.GetSlot());
```

Comment:
this assumes that the two `GetSlot()` are called from left to right, I'm not sure that's guaranteed?